myproject.iml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`myproject.iml` file also should be removed (because it's configuration of your editor for your unique environment). But because name of the project can be different from user to user let's ignore this file not by name, but by extension: `*.iml`